cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -691,7 +691,8 @@ protected int resumeSchedule(String namespace, String appName, String schedule)
     String versionedUrl = getVersionedAPIPath(schedulesUrl, Constants.Gateway.API_VERSION_3_TOKEN, namespace);
     HttpResponse response = doGet(versionedUrl);
     String json = EntityUtils.toString(response.getEntity());
-    return GSON.fromJson(json, new TypeToken<List<ScheduleSpecification>>() { }.getType());
+    return GSON.fromJson(json, new TypeToken<List<ScheduleSpecification>>() {
+    }.getType());
```

Comment:
For some reason intellij insists on re-formatting lines that I did not touch. I will fix this.